types/is-string/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for is-string 1.0
+// Project: https://github.com/inspect-js/is-string
+// Definitions by: aaronleopold <https://github.com/aaronleopold>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function isString(value: any): boolean;
```

Comment:
```suggestion
declare function isString(value: any): value is string;
```
you can mark this function as type predicate:
https://dev.to/daveturissini/aha-understanding-typescript-s-type-predicates-40ha#:~:text=Type%20predicates%20are%20a%20special%20return%20type%20that,Type%20predicates%20are%20expressed%20as%20argumentName%20is%20Type.
It won't affect outcome, but will signal better what's the nature of the function here

Reply:
Oh thank you for linking the article, I've never seen that before!